src/Tasks/Microsoft.WinFx.targets

```diff
@@ -9,6 +9,6 @@
         the shim will internally simply redirect to the real copy of the targets file in the .NET Framework. 
    -->
 
-   <Import Project="$(MSBuildFrameworkToolsPath)\Microsoft.WinFx.targets" Condition="Exists('$(MSBuildFrameworkToolsPath)\Microsoft.WinFx.targets')" />
+   <Import Project="$(MSBuildFrameworkToolsPath)\Microsoft.WinFx.targets" Condition="'$(ImportWinFXTargets)' != 'false' and Exists('$(MSBuildFrameworkToolsPath)\Microsoft.WinFx.targets')" />
```

Comment:
Why not use `ImportInboxWinFXTargets` or `ImportFrameworkWinFXTargets`, seems more clear as to what we're not importing and where it's from. I prefer the former name.

Reply:
Good suggestion. I opted for ImportFrameworkWinFXTargets. 

Thanks!